targets/mbedk64f/source/jerry_extapi.cpp

```diff
@@ -14,19 +14,15 @@
  */
 
 #include "jerry-core/jerry.h"
-
 #include <stdlib.h>
 #include <stdio.h>
-
```

Comment:
These two empty lines help to separate the include types, such as jerry API, system and target headers. Why did you remove them?